lib/commands/view.js

```diff
@@ -236,6 +236,15 @@ class View extends BaseCommand {
       }
     })
 
+    // No data has been pushed because no data is matching the specified version
+    if (data.length === 0 && version !== 'latest') {
```

Comment:
is an asterisk normalized to latest?

Reply:
No, but this bug isn't currently present if you query for `pkg@*` you already get a proper error.